src/features/events_list/index.tsx

```diff
@@ -1,16 +1,16 @@
 import { useAction, useAtom } from '@reatom/react';
 import { useCallback } from 'react';
 import { currentEventAtom } from '~core/shared_state';
-import { scheduledAutoFocus } from '~core/shared_state/currentEvent';
+import { currentEventBbox } from '~features/events_list/atoms/currentEventBbox';
```

Comment:
import inside module should be relative  
`import { currentEventBbox } from './atoms/currentEventBbox';`